my-app/components/labeled-field.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
+import { Text, TextInput, View } from "react-native";
+
+interface LabeledFieldProps {
+  label: string;
+}
+
+export function LabeledField(props: LabeledFieldProps) {
+  return (
+    <View
+      style={{
+        gap: 8,
+      }}
+    >
```

Comment:
Nit: quando forem poucos campos de style, dá pra deixar inline mesmo, acredito que facilite a leitura

```suggestion
    <View style={{ gap: 8}}>
```